contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--1.2.1--1.3.0.sql

```diff
@@ -0,0 +1,7 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
+\echo Use "ALTER EXTENSION ""babelfishpg_tsql"" UPDATE TO '1.3.0'" to load this file. \quit
+ 
+SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+ 
```

Comment:
Shouldn't we make the changes for babelfish_drop_deprecated_object procedure?

Reply:
It is not needed for current changes.